crates/sqlexec/src/parser.rs

```diff
@@ -1373,6 +1400,20 @@ mod tests {
         }
     }
 
+    #[test]
+    fn copy_to_partition_by() {
+        let test_cases = ["COPY table TO './dest' FORMAT parquet PARTITION BY (year, quarter)"];
+
+        for test_case in test_cases {
+            let stmt = CustomParser::parse_sql(test_case)
+                .unwrap()
+                .pop_front()
+                .unwrap();
+            assert_eq!(test_case, stmt.to_string().as_str());
+        }
+    }
```

Comment:
can we test the error conditions:
- `PARTITION` without `BY`
- an unsupported format?
- an unknown format?